combustion-android-ble/src/main/AndroidManifest.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    package="inc.combustion.framework">
```

Comment:
why did this need to be moved out? just out of curiosity.

Reply:
I think that was a miss--thanks for catching it!